Adding more tests for DataLake

begoldsm opened the pull request:
Adding ACL tests for filesystem
Adding Catalog tests
Renaming files to ensure everything is DataLake*

Commit:
Test Update for DataLake*

Updating the published test package to point to our official DNS
suffixes.
Updating all test recordings to run against production
Enable Storage account tests now that we run against production.

Commit:
Update DataLake tests

Add tests for catalog
Add test for ACLs

azurecla commented:
Hi __@begoldsm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (begoldsm). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
make sure the tests are fully up to date.

For some reason it was looking for a different folder to exist. Trying
again with a latest run of the functionality.

Commit:
update to ensure creation of two test folders

Commit:
Update the tests and include them

Missed including new json test recordings for new tests in project. Also
renamed them to be in line with other test naming conventions.

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

stankovski commented on src/ResourceManagement/DataLake.AnalyticsCatalog/DataLakeAnalyticsCatalog.Tests/Properties/AssemblyInfo.cs:
```diff
@@ -1,15 +1,14 @@
 ﻿using System.Reflection;
```
Copyright header

Commit:
Adding copyright header

Missed header on new assemblyInfo file.

stankovski commented:
@yugangw-msft please merge when you are ready

stankovski commented:
@begoldsm please pull from upstream and update the PR

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
DataLake updates

Update to catalog to include Procs, views, types, get seccret and
credential
Update filesystem to remove internal calls
Update version numbers in user agent to be accurate.

begoldsm commented:
I am closing this PR as we are re-opening it in the master branch.
